### DISCUSSION TITLE
用AI科技為一片湛藍海洋，守護水下的生態美

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#695] [General]
海科館每年都會進行珊瑚的健康體健，但由於不同團隊對水下生物的辨識能力不同，在資料搜集上都會出現一些瓶頸與難題，因此國內新創團隊台灣智能海洋No.1研發了水下生物辨識的相關技術。

他們先在潮境公園設了水下攝影機，錄製十幾個小時的影片，並還原水下色彩，建立了龐大的魚群與海底生物資料庫。
以後海科館或任何團隊在進行水下探勘時，利用他們的辨識技術都能更快速更順利，為台灣的海洋保育盡一份心力！

我自己是很喜歡衝浪跟潛水的人，好希望我未來的工作能與我現在所學還有我所愛的大海結合喔！

文章來源：https://www.thenewslens.com/feature/aishow/176582

**Comment by titan92601** [comment]
感覺很不錯欸，即可以環保又可以節省很多時間、人力和財力，一舉數得。我們在AI面前只是螻蟻而已，但我們創造了AI。說來真是諷刺身為創作者既然不如生產品～～

**Comment by Vincent20030314** [comment]
AI不僅能用在產業上，也能用在環境保育上，為生態保育保盡一份力，聽起來很不錯。
